### PR TITLE
Made Redis location configurable.

### LINKED ISSUE #14: Section 4 - .NET example code only connects to local Redis
In section 4, the .NET example code only connects to Redis on `localhost:6379`.  Update the HTML document and source code to have a configurable Redis connection in the same way that section 2 does.

### PR DESCRIPTION
**Pull request by simonprickett**
Closes #14 - makes the Redis URL configurable so that this example code can be run against a cloud instance of Redis.

### CODE REVIEW

File: src/dotnet/redis_om_search_example/Program.cs [hunk 1]
```diff
@@ -9,7 +9,8 @@
 }
 
 var action = args[0];
-var provider = new RedisConnectionProvider("redis://localhost:6379");
+var url = Environment.GetEnvironmentVariable("REDIS_OM_URL") ?? "redis://localhost:6379";
```
Comment (slorello89) [inline src/dotnet/redis_om_search_example/Program.cs:12]:
```suggestion
var uri = Environment.GetEnvironmentVariable("REDIS_OM_URL") ?? "redis://localhost:6379";

```

File: src/dotnet/redis_om_search_example/Program.cs [hunk 2]
```diff
@@ -9,7 +9,8 @@
 }
 
 var action = args[0];
-var provider = new RedisConnectionProvider("redis://localhost:6379");
+var url = Environment.GetEnvironmentVariable("REDIS_OM_URL") ?? "redis://localhost:6379";
+var provider = new RedisConnectionProvider(url);
```
Comment (slorello89) [inline src/dotnet/redis_om_search_example/Program.cs:13]:
```suggestion
var provider = new RedisConnectionProvider(uri);

```

**Review by slorello89 — CHANGES_REQUESTED** [review]
just need to change url -> uri

**Comment by simonprickett** [comment]
@slorello89 I'll do same change here for consistency? https://github.com/redislabs-training/ru204/blob/main/src/dotnet/redis_om_example/Program.cs

**Comment by slorello89** [comment]
@simonprickett - yep that would make sense.

**Pull request merged**